### DISCUSSION TITLE
[Pimcore X] Sent mails do not appear under "Sent Emails"

### DISCUSSION THREAD

**Discussion by akno77** [pimcore/pimcore#10068] [Q&A / Support]
Hi,

when I send mails via email document and code like this:

`$mail = new \Pimcore\Mail();`
`$mail->setDocument($params['mailTemplate']);`
`$mail->setParams($params);`
`$mail->setIgnoreDebugMode(true);`
`$mail->enableLogging();`
`$mail->send();`

The mail is sent correctly, but it does not pop up in the Tab "Sent Emails". Is this a current bug or do I miss sth.?

Does it make a difference if I am in dev mode or not? I tried it also with "prod" mode, but same result...

**Comment by dvesh3** [comment]
sorry, not reproducible on latest version 10.1.1.

**Comment by akno77** [comment]
I still have the problem, that sent mails are not displayed in the "Sent Emails" tab of the Mail Document. Under "Tools -> Email -> Sent Emails" they appear.

I use the mail document like this:

```
$mail = new \Pimcore\Mail();
$mail->setDocument($params['mailTemplate']);
$mail->setParams($params);
$mail->setIgnoreDebugMode(true);
$mail->enableLogging();
$mail->send();
```

I see in the db table "email_log" that the field "documentId" is null.
What could be the problem?

I use Pimcore 10.1.3

**Reply by fashxp** [reply]
see https://github.com/pimcore/pimcore/issues/10354
